templates/flannel.go

```diff
@@ -74,6 +74,7 @@ data:
   net-conf.json: |
     {
       "Network": "{{.ClusterCIDR}}",
+      "SubnetLen": {{ .FlannelBackend.SubnetLen }},
```

Comment:
There are 4 canal templates, and 2 flannel templates. I think this should go into all of them if it's supported by the addon versions.

Reply:
https://github.com/rancher/rke/blob/master/templates/templates.go#L17-L25

Reply:
Thanks for your review. Yes I'm gonna add this to all of templates. 
Could you check PR again?

Reply:
@moelsayed